### DISCUSSION TITLE
Sync job from SQL Server to Apache Doris

### DISCUSSION THREAD

**Discussion by rajeshguptaiiitm** [apache/doris#37676] [A - General / Q&A] [answered]
Hi All,
I am new to Doris and evaluating the Doris for one of our project. I want to sync SQL Server Data to Apache Doris with CDC.
How can I achieve it ? Is there any documentation or steps to follow it.

**Comment by zy-kkk** [comment, marked as answer]
You can refer to the following document:
https://doris.apache.org/docs/ecosystem/flink-doris-connector/#use-flinkcdc-to-access-multiple-tables-or-the-entire-database-supports-mysql-oracle-postgresql-sqlserver
